<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatbox</class>
 <widget class="QWidget" name="chatbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>660</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>601</width>
    <height>448</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1001</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btn_min{
	border:none;
	image: url(:/res/zuixiaohua.png);
}

QPushButton#btn_close{
	border:none;
	image: url(:/res/guanbi.png);
}
QPushButton#btn_min:hover{
	background-color: rgba(190 ,190 ,190 ,100);
	
}
QPushButton#btn_close:hover{
	background-color: rgb(255, 0, 0)
}
QPushButton#btn_color{
	border:none;
}
QPushButton#btn_clear{
	border:none;
}
QPushButton#btn_other{
	border:none;
}
QLabel#label_top{
background-color:rgb(0, 170, 255)
}
QPushButton#btn_more{
	border:none;
	image: url(:/res/more.png);
}
QPushButton#btn_more:hover{
	background-color: rgba(190 ,190 ,190 ,100);
}
QPushButton#btn_send{
	border:none;
	background-color: rgb(18,193,245);
	border-radius:5px;
	color:white;
}
QWidget#widget{
	border:none;
	background-color:white;
}
QWidget#widget_2{
 background-color:  rgb(240,240,240); border-top-color: gray; border-bottom-color: gray; border-left-color: transparent; border-right-color: transparent;border-width:10px
}
QPushButton#btn_folder{
	border:none;
	
	image: url(:/res/folder.png);
}</string>
  </property>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>522</y>
     <width>823</width>
     <height>138</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>625</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_top">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>821</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_friend_username">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/mainTitle.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_min">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/res/最小化.png</normaloff>:/res/最小化.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/res/关闭2.png</normaloff>:/res/关闭2.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_more">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>482</y>
     <width>821</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_pahoto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/res/photos.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_folder">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>40</y>
     <width>821</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
